modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6842,7 +6843,7 @@ favorites=Favorites
 fax=Fax
 fax-number=Fax Number
 fbml=FBML
-fds-cell-renderer=Frontend Data Set Cell Renderer
+fds-cell-renderer=Cell Renderer
```

Comment:
This language key should be `cell-renderer`.

Reply:
If these changes are strictly necessary I'll do it after the confirmation of Dani on whether we need to keep the "FDS" in the names or not.